spikes/data/dataservice/app/DataAccessLayer/InMemoryDal.cs

```diff
@@ -85,12 +81,12 @@ public InMemoryDal()
             }
 
             // load the data from the json file
```

Comment:
converted to system.text.json - dynamic doesn't work so created a model class

issue on the board to fix naming - it has dependencies